server/src/main/java/CSC4410/CovidTracker/Config.java

```diff
@@ -0,0 +1,48 @@
+package CSC4410.CovidTracker;
+
+import CSC4410.CovidTracker.util.Filesystem;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Application configuration, loaded as JSON from `Config.configFile`.
+ */
+public class Config {
+
+    private static final String configFile = "config.json";
+
+    static {
+        try {
+            loadConfig();
+        } catch (IOException | JSONException e) {
+            e.printStackTrace();
+            System.exit(1);
+        }
+    }
+
+    private static void loadConfig() throws IOException, JSONException {
+        var content = Filesystem.readFile(configFile);
```

Comment:
var is a good feature that Java 15 provides but i recommend specifying the data type for a better understanding of the variables